ui/apps/platform/cypress/integration/vulnerabilities/exceptionManagement/pendingRequestsTable.test.js

```diff
@@ -0,0 +1,94 @@
+import withAuth from '../../../helpers/basicAuth';
+import { hasFeatureFlag } from '../../../helpers/features';
+import {
+    cancelAllCveExceptions,
+    fillAndSubmitExceptionForm,
+    selectSingleCveForException,
+    verifyExceptionConfirmationDetails,
+    verifySelectedCvesInModal,
+    visitWorkloadCveOverview,
+} from '../workloadCves/WorkloadCves.helpers';
+import { selectors as workloadCVESelectors } from '../workloadCves/WorkloadCves.selectors';
+import { visitExceptionManagement } from './ExceptionManagement.helpers';
+
+describe('Exception Management Pending Requests Page', () => {
+    withAuth();
+
+    before(function () {
+        if (
+            !hasFeatureFlag('ROX_VULN_MGMT_WORKLOAD_CVES') ||
+            !hasFeatureFlag('ROX_VULN_MGMT_UNIFIED_CVE_DEFERRAL') ||
+            !hasFeatureFlag('ROX_WORKLOAD_CVES_FIXABILITY_FILTERS')
+        ) {
+            this.skip();
+        }
+    });
+
+    beforeEach(() => {
+        if (
+            hasFeatureFlag('ROX_VULN_MGMT_WORKLOAD_CVES') &&
+            hasFeatureFlag('ROX_VULN_MGMT_UNIFIED_CVE_DEFERRAL') &&
+            hasFeatureFlag('ROX_WORKLOAD_CVES_FIXABILITY_FILTERS')
+        ) {
+            cancelAllCveExceptions();
+        }
+    });
+
+    after(() => {
+        if (
+            hasFeatureFlag('ROX_VULN_MGMT_WORKLOAD_CVES') &&
+            hasFeatureFlag('ROX_VULN_MGMT_UNIFIED_CVE_DEFERRAL') &&
+            hasFeatureFlag('ROX_WORKLOAD_CVES_FIXABILITY_FILTERS')
+        ) {
+            cancelAllCveExceptions();
+        }
+    });
+
+    it('should be able to view deferred pending requests', () => {
+        visitWorkloadCveOverview();
+
+        // defer a single cve
+        selectSingleCveForException('DEFERRAL').then((cveName) => {
+            verifySelectedCvesInModal([cveName]);
+            fillAndSubmitExceptionForm({
+                comment: 'Test comment',
+                expiryLabel: 'When all CVEs are fixable',
+            });
+            verifyExceptionConfirmationDetails({
+                expectedAction: 'Deferral',
+                cves: [cveName],
+                scope: 'All images',
+                expiry: 'When all CVEs are fixable',
+            });
+
+            visitExceptionManagement();
+
+            // the deferred request should be pending
+            cy.get(
+                'table td[data-label="Requested action"]:contains("Deferred (when all fixed)")'
```

Comment:
Since we're dealing with one exception at a time, I think this is fine for now. I can imagine improving the selection by doing the following:
1. Add the request name as part of the confirmation modal
2. Use the request name value from the confirmation modal to find the right row in the pending requests table (by request name of course)
3. Check to see if the values are correct